test/integration/httpServer.test.js

```diff
@@ -19,4 +19,29 @@ describe('health', function () {
     expect(actualResult.status).to.equal(200)
     expect(actualResult.body).to.deep.equal(expectedResult)
   })
+
+  test('POST Order - 201', async function () {
+    const newProject = {
+      owner: 'BAE',
+      manufacturer: 'Maher',
+      status: 'Accepted',
+      requiredBy: new Date().toISOString(),
+      items: [],
```

Comment:
will checking the `id`s in `items` against the `recipe` db be in a later PR?

Reply:
Yeah, this was just to provide the basics for now